test/requirements.txt

```diff
@@ -4,3 +4,5 @@ six
 coverage
 codecov
 mock
+pytest
+asynctest
```

Comment:
Don't see `pytest` usage. `asynctest` is async tests dependency only.

Reply:
The file isn't used now. Removed both of lines.